logging_utilities/formatters/json_formatter.py

```diff
@@ -343,18 +343,16 @@ def format(self, record):
         if self.add_always_extra:
             extra = self._get_extra_attrs(record)
 
-        message = self.formatMessage(record)
-
-        if self.add_always_extra:
-            self._add_extra_to_message(extra, message)
-
         if record.exc_info:
             # Cache the traceback text to avoid converting it multiple times
             # (it's constant anyway)
             if not record.exc_text:
                 record.exc_text = self.formatException(record.exc_info)
-        if record.exc_text:
-            message['exc_text'] = record.exc_text
+
+        message = self.formatMessage(record)
+
+        if self.add_always_extra:
+            self._add_extra_to_message(extra, message)
 
         if record.stack_info:
```

Comment:
I'm not sure I understand exactly what was the issue but it seems that `stack_info` is setup the same as `exc_text` so we might have the same issue ?

Reply:
As discussed in person and updated in the PR's description